### PR TITLE
docs(interact-with-data): add projectURL to js sdk init config

### PR DESCRIPTION
**Pull request by marlosin**

### CODE REVIEW

File: docs/tools/README.md [hunk 1]
```diff
@@ -174,7 +183,7 @@ and then do similar commands to get data. Below you can find example:
     if (!jexia) {
       throw new Error("Please inform support team of Jexia.");
     }
-    //Initialize dataOperationsModule
+    // Initialize dataOperationsModule
     const dataModule = jexia.dataOperations();
     const credentials = {
       projectID: "<your-project-id>",
```
Comment (MishaJexia) [inline docs/tools/README.md:189]: Shell we have here project URL?

Comment (marlosin) [reply docs/tools/README.md:189]: How do you suggest we add it? providing `projectURL` would make `projectID` and `zone` irrelevant, it's a choice for the user. Should we add code here for both ways?

Comment (MishaJexia) [reply docs/tools/README.md:189]: You mean I can use ProjectURL or ProjectId + Project zone?

Comment (marlosin) [reply docs/tools/README.md:189]: yes, that's said some lines above https://github.com/jexia/docs/pull/41/files#diff-775b3d836b5b90f2b70daba2c3d4f0e5R58

**Pull request merged**